src/network/core.rs

```diff
@@ -275,6 +290,20 @@ impl Core {
             None => 0,
         };
 
+        let isolation_config = match network.options.as_ref().and_then(|map| map.get("isolate")) {
+            Some(isolation) => match isolation.parse() {
+                Ok(isolation) => isolation,
+                Err(err) => {
+                    return Err(std::io::Error::new(
+                        std::io::ErrorKind::Other,
+                        format!("unable to parse isolation type: {}", err),
+                    ))
+                }
+            },
+            // default is to not isolate podman networks
+            None => false,
+        };
+
```

Comment:
This is not relevant, isolation should only be used with the bridge driver.

Reply:
Ok I was not sure which to put it in since you linked the MTU example to macvlan above, I will remove this